datasetinsights/datasets/synthetic.py

```diff
@@ -72,3 +85,182 @@ def read_bounding_box_2d(annotation, label_mappings=None):
         bboxes.append(box)
 
     return bboxes
+
+
+class SynDetection2D:
```

Comment:
@86sanj This looks like an object that will confuse users with the pytorch dataset class. Can we support visual inspection in the `preception_statistics` notebook without this class?